docs/src/Fieldset.doc.js

```diff
@@ -0,0 +1,222 @@
+// @flow strict
+import type { Node } from 'react';
+import { Fieldset } from 'gestalt';
+import PropTable from './components/PropTable.js';
+import PageHeader from './components/PageHeader.js';
+import MainSection from './components/MainSection.js';
+
+const cards: Array<Node> = [];
+const card = (c) => cards.push(c);
+
+card(
+  <PageHeader
+    name="Fieldset"
+    description="Creates an accessible fieldset and legend for a group of related form items"
+    defaultCode={`
+      function RadioButtonExample() {
+        const [favorite, setFavorite] = React.useState(undefined);
+
+        return (
+          <Fieldset legend="What is your favorite pet?">
```

Comment:
![image](https://user-images.githubusercontent.com/10593890/117052693-074eea80-ace6-11eb-81d0-953d88df5fe7.png)

Maybe something more businessy?

Reply:
Kept the top one fun, but made the others more realistic